identity-service/src/routes/cognito.js

```diff
@@ -105,6 +106,14 @@ module.exports = function (app) {
           // deduping against lowercased names
           identities.push(JSON.stringify({ dob, name: nameLowercased }))
         }
+        if (documentVerification && documentVerification.status === 'success') {
+          // if document verification is not null, then status and documents are always present
+          // within each document, the status is always present, and the extracted_data is always present but nullable
+          const successfulExtractedDataList = documentVerification.documents
+            .filter(document => document.status === 'success' && document.extracted_data)
+            .map(document => JSON.stringify(document.extracted_data))
+          successfulExtractedDataList.forEach(item => identities.push(item))
```

Comment:
Do we want the whole extracted data? Don't we just want the `id_number`? Otherwise we're opening ourselves up to more potentially altered fields, etc